### ISSUE TITLE
Kp, Dst, F10.7 and IMF

### ISSUE THREAD

**Issue by lmar76** [ESA-VirES/VirES-Python-Client#44]
As already done for data downloaded from the VirES web interface, we should disable the download of Kp, Dst, F10.7 and IMF indices via Python client.

**Comment by smithara** [comment]
Is this because of the non-ESA sources of the data? The access to these indices through viresclient is quite useful for some workflows (I use it myself).

Perhaps instead of removing them, we can add some source acknowledgement with links to the responsible organisations. This might be as metadata in the data objects created and/or as an info message at run time when these data are accessed.

**Comment by lmar76** [comment]
Yes, we are trying to understand if it is possible to redistribute this data. For the time being, please put this issue on hold.

**Comment by smithara** [comment]
These data are available (redistributed) through a number of other channels, perhaps most notably by the NASA OMNI service (https://omniweb.gsfc.nasa.gov/html/ow_data.html#3) and there are other Python packages out there (e.g. pysat) that provide access. I have no idea of the legal issues about ESA providing access to these, but the data are all publicly available.

AFAIK the IMF data are all originally sourced from NASA (spacecraft upstream from Earth) - the OMNI service has some description of the contributing spacecraft and processing method. I don't recall the exact origin of the IMF data as part of the Swarm AUX - it may well be via OMNI.

The magnetic indices (Kp, Dst) are organised through IAGA and I think the actual responsible institutions for distributing them have changed over time, though the contributing measurements come from a number of observatories in different countries. For info and citations: see:
- Dst (WDC for Geomagnetism - Kyoto) - http://isgi.unistra.fr/indices_dst.php
- Kp (GFZ Potsdam) - http://isgi.unistra.fr/indices_kp.php

F10.7 comes from Space Weather Canada - https://www.spaceweather.gc.ca/solarflux/sx-en.php
